src/Elmish.WPF/BindingVmHelpers.fs

```diff
@@ -150,50 +150,50 @@ type SubModelSelectedItemBinding<'model, 'msg, 'bindingModel, 'bindingMsg, 'bind
     d.Set id model
 
 
-type BaseVmBinding<'model, 'msg> =
-  | OneWay of OneWayBinding<'model, obj>
-  | OneWayToSource of OneWayToSourceBinding<'model, obj>
+type BaseVmBinding<'model, 'msg, 'a> =
+  | OneWay of OneWayBinding<'model, 'a>
+  | OneWayToSource of OneWayToSourceBinding<'model, 'a>
   | OneWaySeq of OneWaySeqBinding<'model, obj, obj>
-  | TwoWay of TwoWayBinding<'model, obj>
+  | TwoWay of TwoWayBinding<'model, 'a>
   | Cmd of cmd: Command
-  | SubModel of SubModelBinding<'model, 'msg, obj, obj, obj>
-  | SubModelWin of SubModelWinBinding<'model, 'msg, obj, obj, obj>
+  | SubModel of SubModelBinding<'model, 'msg, obj, obj, 'a>
+  | SubModelWin of SubModelWinBinding<'model, 'msg, obj, obj, 'a>
   | SubModelSeqUnkeyed of SubModelSeqUnkeyedBinding<'model, 'msg, obj, obj, obj>
   | SubModelSeqKeyed of SubModelSeqKeyedBinding<'model, 'msg, obj, obj, obj, obj>
-  | SubModelSelectedItem of SubModelSelectedItemBinding<'model, 'msg, obj, obj, obj, obj>
+  | SubModelSelectedItem of SubModelSelectedItemBinding<'model, 'msg, obj, obj, 'a, obj>
 
 
 type CachedBinding<'model, 'msg, 'value> = {
-  Binding: VmBinding<'model, 'msg>
+  Binding: VmBinding<'model, 'msg, 'value>
   Cache: 'value option ref
```

Comment:
Rename `'value` to `'a`

Reply:
Actually I don't think that this cache effect is the correct type. It's its own thing.

Reply:
Nevermind, that is handled elsewhere